apps/openmw/mwgui/inventorywindow.cpp

```diff
@@ -71,7 +71,7 @@ namespace MWGui
         , mUpdateTimer(0.f)
     {
         float uiScale = Settings::Manager::getFloat("scaling factor", "GUI");
-        if (uiScale > 1.0)
+        if (uiScale > 0.f)
```

Comment:
I don't know if it is intended but here you allow down scaling which was not the case before

Reply:
It was the case before, but not every widget supported it due to different bound checks. Docs says that UI scaling factor should be >0, not >= 1.